.gitignore

```diff
@@ -15,4 +15,7 @@ docs/
 build/*
 !build/.gitkeep
 _install/
-bin/
+
+!.gitkeep

```

Comment:
Maybe we just do `!*.gitkeep`? Does that work the same as what we have?

Reply:
I think so